### ISSUE TITLE
Github Flavoured Markdown formatting issues on 3.13

### ISSUE THREAD

**Issue by hanxue** [gitbucket/gitbucket#1171]
## Markdown documents not rendered correctly
**Impacted version**: 3.13

**Deployment mode**: Running on jetty, on Windows 7

## Problem description:
When the [AirBnB Javascript Guide markdown document](https://github.com/airbnb/javascript/blob/master/README.md) is rendered on GitBucket, I see the following errors:

#### 1. `<a></a>` links not rendered correctly. 

*Markdown code*: 
```markdown
<a name="types--primitives"></a><a name="1.1"></a>
  - [1.1](#types--primitives) **Primitives**:
```
*Expected result*: 
<a name="types--primitives"></a><a name="1.1"></a>
  - [1.1](#types--primitives) **Primitives**:

*Actual result*: 
![image](https://cloud.githubusercontent.com/assets/760008/14451267/576e1a98-00b8-11e6-8126-830114e5acd2.png)

#### 2. Code block after blockquote does not get rendered corrected

*Markdown code*: 
```markdown
    > Why? They allow you to define all the properties of an object in one place.

    ```javascript

    function getKey(k) {
      return `a key named ${k}`;
    }
    ```
```
*Expected result*: 

> Why? They allow you to define all the properties of an object in one place.

```javascript

function getKey(k) {
  return `a key named ${k}`;
}
```

*Actual result*: 
![image](https://cloud.githubusercontent.com/assets/760008/14451570/baae626e-00ba-11e6-9145-fb8483081d70.png)

**Comment by takezoe** [comment]
@hanxue Thanks for your feedback. Both of them are [markedj](https://github.com/gitbucket/markedj) problem.

> 1. <a></a> links not rendered correctly.

This is constraint in current markedj implementation by security reason. markedj can't filter danger HTML markup. But in the future, we would like to implement safety HTML rendering: https://github.com/gitbucket/markedj/issues/1

> 2. Code block after blockquote does not get rendered corrected

Fixed in https://github.com/gitbucket/markedj/pull/10. We will merge it after enough testing.

**Comment by hanxue** [comment]
Thank you very much for your update @takezoe

**Comment by takezoe** [comment]
**Code block after blockquote** is fixed in GitBucket 3.14 and 4.0.
